app/main.py

```
@@ -1,14 +1,15 @@
 from contextlib import asynccontextmanager
-from urllib.request import Request
 from fastapi import FastAPI, HTTPException, status
 from fastapi.exceptions import RequestValidationError
 from fastapi.middleware.cors import CORSMiddleware
-from fastapi.responses import JSONResponse
 
 from app.router import api_router
 from app.API.Utilities.exception_handlers import unprocessable_entity_handler, http_exception_handler, generic_exception_handler
+from app.Core.AI.registration import register_ai_providers
```

Ruff (0.11.9)

`app.Core.AI.registration.register_ai_providers` imported but unused

Remove unused import: `app.Core.AI.registration.register_ai_providers`

(F401)